src/models/data.py

```diff
@@ -148,13 +157,15 @@ def __init__(self, loader: DataLoader) -> None:
         self.clear_nans = loader.clear_nans
         self.surrounding_pixels = loader.surrounding_pixels
         self.to_tensor = loader.to_tensor
+        self.experiment = loader.experiment
 
         if self.shuffle:
             # makes sure they are shuffled every epoch
             shuffle(self.data_files)
 
         self.normalizing_dict = loader.normalizing_dict
         self.normalizing_array: Optional[Dict[str, np.ndarray]] = None
+        self.current_normalizing_array: Optional[Dict[str, np.ndarray]] = None
```

Comment:
It looks like the `current_normalizing_array` isn't used. Can it be deleted (from here and from line 189)?

Reply:
yup! good spot